test/keyboard-input.html

```diff
@@ -517,6 +518,90 @@
             expect(result.getDate()).to.equal(9);
           });
 
+          it('should throw when passing a year < 0', () => {
+            expect(() => Vaadin.DatePickerHelper._getAdjustedYear(today, -1, 1, 1)).to.throw(Error);
+          });
+
+          it('should throw when passing a year >= 100', () => {
+            expect(() => Vaadin.DatePickerHelper._getAdjustedYear(today, 100, 1, 1)).to.throw(Error);
+          });
+
+          it('should parse with default day value when day is not provided', () => {
+            expect(Vaadin.DatePickerHelper._getAdjustedYear(new Date(1919, 11, 31), 70, 0)).to.equal(1870);
```

Comment:
Seems like a duplicate line.
```suggestion
```

Reply:
Fixed it. Thanks. Is anything else worth mentioning or fixing?
(Asynch test is a bit harder to work out, and fix, not sure if needed if those are needed, if not I can just delete the lines...)

Reply:
Thanks, I will have a look into this and try to make it work it! 🙇